css/style.css

```diff
@@ -1,5 +1,5 @@
 html {
-    font-family: Comic Sans MS;
+    font-family: Comic Sans MS, sans-serif;
```

Comment:
no. <3

Reply:
I understand now, let me revert this mistake